def handler(event, context):
    raise Exception('Exception occured')
